### ISSUE TITLE
ResNet question

### ISSUE THREAD

**Issue by dkoguciuk** [JirongZhang/DeepHomography#16]
Hi @JirongZhang ,

I've seen you've updated the paper on ArXiv. It looks even better than before, good job!

Do you use pretrained ResNet34 weights or you're training from scratch? Are your feature extraction and mask estimation models using the same ResNet-block as the ResNet34 backbone or they use standard conv2d?

Best,
Daniel

**Comment by JirongZhang** [comment]
@dkoguciuk , Thanks a lot. 
1.  Due to min(|I-H_ab*H_ba|^2) is difficult, it is not allowed to initialize randomly, so we suggest to use the pre-trained model from Pytorch(https://download.pytorch.org/models/resnet34-333f7ec4.pth) for initialization. In addition, only predicting H_ab is called "Oneline". Compared with the "Doubleline" which predicts H_ab & H_ba simultaneously has less cost.  If you use the "Oneline" model a to train, you don't need to calculate |I-H_ab*H_ba|^2，then you can training from scratch.
2. They are standard conv2d. The last activation function of mask estimation model is sigmoid.
3. Our open source plan has been delayed, but will be completed in the next few days.

**Comment by dkoguciuk** [comment]
@JirongZhang , makes absolutely sense. Thank you for your reponse!

Best,
Daniel
